src/components/modules/nutrition/add-food/food-list-item-info.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div v-if="foodNutrients">
+        <div class="food-info">
+            <div>
+                <span>Protein: </span>
+                <span>{{ foodNutrients.protein.value * portion | cutDigits}}{{ foodNutrients.protein.unit }}</span>
+            </div>
+            <div>
+                <span>Fat: </span>
+                <span>{{ foodNutrients.fat.value * portion | cutDigits }}{{ foodNutrients.fat.unit }}</span>
+            </div>
+            <div>
+                <span>Carbs: </span>
+                <span>{{ foodNutrients.carbs.value * portion | cutDigits }}{{ foodNutrients.carbs.unit }}</span>
+            </div>
+        </div>
+        <v-form v-if="showMore" class="portion-form">
+            <v-text-field
+                v-model="portion"
+                label="Number of portions"
+                type="number"
+            ></v-text-field>
```

Comment:
I prefer leaving closing `>` on the same line as last attribute, this just looks weird for me ;)

But that's a matter of convention of course.

Reply:
Yeah, I've seen this convention somewhere else and I copied this style from there

Reply:
Done